### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Microverse_Portfolio: Accessibility check

In this PR I and my coding partener :
* Checked the accessibility of my portfolio 

List of checked and actions taken:
  - Page titles - no issues found
  - Image text alternatives  -  Add alternate text on social icons images 
  - Text headings  -  adjust the heading element in a sequentially-descending order
  - Color contrast  -  fix backgorund and foreground colors
  - Resize  -  no issues found
  - Interaction  -  no issues found
  - Moving content  -  no issues found
  - Multimedia  -  no issues found
  - The basic structure of the page  -  no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,271 +2,306 @@
...
-    <link rel="shortcut icon" href="img/microverse-logo.png">
-    <style>
-
-    </style>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="description"
+    content="This is a portfolio work fro microverse module 1 template one which student use to learn more about html git and github and best practises">
+  <title>Microverse Portfolio</title>
```
Comment (vechicin) [inline index.html]: According to the project requirement, you should implement an adequate title for your project.

Screen Shot 2022-06-07 at 11 05 49

Kindly fix this.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @ChrispaixK and @markotuya0 ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good PR title/description
- [x] Good job overall implementing the accessibility features

## Required Changes ♻️

- [ ] As part of the evaluation guidelines we are asked to run your live demo through [this resource](https://www.aditus.io/button-contrast-checker/). Some of your checks are not passing, please run your project through it and fix the necessary contrasts. I understand it is strange to be asked to follow a template that is not fully accessible but, unfortunately, that is the crux of the matter. Kindly fix your contrasts.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ChrispaixK) [reply index.html]: Thank you @vechicin

**Review by t-yanick — APPROVED** [review]
Hi @ChrispaixK, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**